### PR TITLE
modify front end of image details

### PR DESCRIPTION
**Pull request by ywk253100**

### CODE REVIEW

File: static/resources/js/components/repository/popup-details.directive.js [hunk 1]
```diff
@@ -35,8 +35,8 @@
     
     function getManifestSuccess(data, status) {
       console.log('Successful get manifest:' + data);
-      vm.manifest = data;      
-      vm.manifest['Created'] = $filter('dateL')(vm.manifest['Created'], 'YYYY-MM-DD HH:mm:ss');
+	  vm.manifest = jQuery.parseJSON(data.manifest.history[0].v1Compatibility);      
+      vm.manifest['created'] = $filter('dateL')(vm.manifest['created'], 'YYYY-MM-DD HH:mm:ss');
```
Comment (wknet123) [inline static/resources/js/components/repository/popup-details.directive.js]: Use 'angular.toJson()' method will be better.

File: static/resources/js/components/repository/popup-details.directive.js [hunk 2]
```diff
@@ -89,21 +89,19 @@
...
           '<div class="form-group"><label class="col-sm-3 control-label">Created</label>' +
-          '<div class="col-sm-9"><p class="form-control-static long-line-margin-right">' + ctrl.manifest['Created'] + '</p></div></div>' +
-          '<div class="form-group"><label class="col-sm-3 control-label">Duration Days</label>' +
-          '<div class="col-sm-9"><p class="form-control-static long-line-margin-right">' + (ctrl.manifest['Duration Days'] === '' ? 'N/A' : ctrl.manifest['Duration Days']) + ' days</p></div></div>' +
+          '<div class="col-sm-9"><p class="form-control-static long-line-margin-right">' + ctrl.manifest['created'] + '</p></div></div>' +
           '<div class="form-group"><label class="col-sm-3 control-label">Author</label>' +
-          '<div class="col-sm-9"><p class="form-control-static long-line-margin-right">' + (ctrl.manifest['Author'] === '' ? 'N/A' : ctrl.manifest['Author']) + '</p></div></div>' +
+          '<div class="col-sm-9"><p class="form-control-static long-line-margin-right">' + ((ctrl.manifest['author'] === ''|| typeof ctrl.manifest['author'] === "undefined") ? 'N/A' : ctrl.manifest['author']) + '</p></div></div>' +
```
Comment (wknet123) [inline static/resources/js/components/repository/popup-details.directive.js]: Try to handle with 'angular.isDefined()' method.

File: static/resources/js/services/repository/services.list-manifest.js [hunk 3]
```diff
@@ -29,7 +29,8 @@
         .get('/api/repositories/manifests', {
           'params': {
             'repo_name': repoName,
-            'tag': tag
+            'tag': tag,
+			'version': 'v1',
```
Comment (wknet123) [inline static/resources/js/services/repository/services.list-manifest.js]: Make the spaces align with last line. (2 spaces per indent)

**Comment by wknet123** [comment]
LGTM

**Comment by reasonerjt** [comment]
build break???

**Pull request merged**